src/TTextEdit.cpp

```diff
@@ -1345,7 +1345,9 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
             // killing the gdb debugger instance run by Qt Creator will restore
             // normality.
             connect(mpContextMenuAnalyser, &QAction::hovered, this, &TTextEdit::slot_analyseSelection);
-            mpContextMenuAnalyser->setToolTip(tr("<p>Hover on this item to display the Unicode codepoints in the selection <i>(only the first line!)</i></p>"));
+            mpContextMenuAnalyser->setToolTip(QStringLiteral("<p>%1</p>").arg(
+                tr("Hover on this item to display the Unicode codepoints in the selection <i>(only the first line!)</i>",
+                    "Codepoint refers to the actual unicode number of a character: https://en.wikipedia.org/wiki/Unicode#Architecture_and_terminology/")));
```

Comment:
That terminal `/` breaks the URL...!
```suggestion
                   "Codepoint refers to the actual unicode number of a character: https://en.wikipedia.org/wiki/Unicode#Architecture_and_terminology")));
```

Reply:
Good catch!